datafeeds/parsers/fms_api/fms_api_team_avatar_parser.py

```diff
@@ -0,0 +1,68 @@
+from google.appengine.ext import ndb
+
+from models.media import Media
+from models.team import Team
+from consts.media_type import MediaType
+from helpers.media_manipulator import MediaManipulator
+
+
+class FMSAPITeamAvatarParser(object):
+    def __init__(self, year):
+        self.year = year
+
+    def parse(self, response):
+        """
+        Parse team avatar from FMSAPI
+        """
+
+        # Get team avatar
+        current_page = response['pageCurrent']
+        total_pages = response['pageTotal']
+        teams = response['teams']
+        ret_models = []
+        media_to_put = []
+
+        for teamData in teams:
+            if teamData['encodedAvatar'] is u'':
```

Comment:
`if not teamData['encodedAvatar']` is probably safer check, since that will be false if it is null or empty.